io.c

```diff
@@ -13331,6 +13331,163 @@ set_LAST_READ_LINE(VALUE val, ID _x, VALUE *_y)
  *    require 'io/console'
  *    rows, columns = $stdout.winsize
  *    puts "Your screen is #{columns} wide and #{rows} tall"
+ *
+ *  == What's Here
+ *
+ *  First, what's elsewhere. \Class \IO includes the module Enumerable,
+ *  which provides dozens of additional methods.
+ *
+ *  Here, class \IO provides methods that are useful for:
+ *
+ *  - {Creating}[#class-IO-label-Creating]
+ *  - {Reading}[#class-IO-label-Reading]
+ *  - {Writing}[#class-IO-label-Writing]
+ *  - {Copying}[#class-IO-label-Copying]
+ *  - {Positioning}[#class-IO-label-Positioning]
+ *  - {Iterating}[#class-IO-label-Iterating]
+ *  - {Converting}[#class-IO-label-Converting]
+ *  - {Settings}[#class-IO-label-Settings]
+ *  - {Querying}[#class-IO-label-Querying]
+ *  - {Closing}[#class-IO-label-Closing]
+ *  - {Encoding}[#class-IO-label-Encoding]
+ *  - {Buffering}[#class-IO-label-Buffering]
+ *  - {Low-Level Access}[#class-IO-label-Low-Level+Access]
+ *
+ *  === Creating
+ *
+ *  - ::new (aliased as ::for_fd):: Creates and returns a new \IO object for the given
+ *                                  integer file descriptor.
+ *  - ::open:: Creates and new \IO object.
+ *  - ::pipe:: Creates a reader \IO object and a writer \IO object, connected.
+ *  - ::popen:: Creates a write \IO object to receive output from the given
+ *              system command.
+ *  - ::select:: Selects a ready \IO object from arrays of \IO objects
+ *               for reading, writing, or error.
+ *
+ *  === Reading
+ *
+ *  - ::binread:: Returns a specified sequence of file bytes for a given filepath,
+ *                read in binary mode.
+ *  - ::read (aliased as ::readlines):: Returns a specified sequence of file bytes
+ *                                      for a given filepath.
+ *  - #advise:: Announces the intention to access data from +self+ in a specific way.
+ *  - #getbyte:: Returns the next 8-bit byte read from +self+.
+ *  - #getc:: Returns the next character read from +self+.
+ *  - #gets:: Returns the line read from +self+.
+ *  - #pread:: Returns a specified sequence of bytes read from +self+,
+ *             not disturbing the current offset.
+ *  - #read:: Returns the next _n_ bytes read from +self+ for a given _n_.
```

Comment:
This change is still needed.

Reply:
This change to include `all or the next _n_` is still needed.  Maybe `all remaining or the next _n_` is even better, since if the current position is not the start of the file, it is more accurate.